### ISSUE TITLE
Error while running prep_data.py

### ISSUE THREAD

**Issue by svknair** [vc1492a/Yelp-Challenge-Dataset#1]
Hi,

While I try to execute the prep_data.py code I get the below error:

UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 2522: character maps to <undefined>

Can someone please offer advice to fix this?

Thanks

**Comment by vc1492a** [comment]
Are you using Python 3? Looks like you may need to specify the encoding when opening the file. This is a common issue when working across different systems. Something like the below should work for you. 

http://stackoverflow.com/questions/30750843/python-3-unicodedecodeerror-charmap-codec-cant-decode-byte-0x9d

This corresponds to lines 34, 172, and 202 in prep_data.py. I'd look at Stack Overflow for any debugging issues you come across next time. There's plenty of responses on there that would help you resolve issues such as this without resorting to the need to open a Github Issue.

**Comment by svknair** [comment]
@vc1492a: Yes, I am using Python 3.6. I think I fixed the issue using the instructions in the SO link that you provided. Thanks for that. But I got a new error:

ValueError: list.remove(x): x not in list

This pertains to line 45 in your code. Have checked SO for tips to handle this, but not able to find the solution. Must admit that I am a novice as far as Python is concerned, so perhaps I am not able to adapt the solutions offered in SO to this specific code. Any help is appreciated.
